torch/_torch_docs.py

```diff
@@ -7327,6 +7327,53 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.vandermonde_matrix,
+           """
+vandermonde_matrix(x, n, increasing=True, dtype=None, \
```

Comment:
Even though we don't have a "default value" for n in the code, conceptually it has the same default as NumPy (None).